psake.psm1

```diff
@@ -621,7 +621,23 @@ function ConfigureBuildEnvironment {
     $frameworkDirs = @()
     if ($buildToolsVersions -ne $null) {
         foreach($ver in $buildToolsVersions) {
-            if (Test-Path "HKLM:\SOFTWARE\Microsoft\MSBuild\ToolsVersions\$ver") {
+            if ($ver -eq "15.0") {
+                $vs2017paths = @(
+                    "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise",

```

Comment:
Because PSake is using the .net version to specify the build tools it should use, Could this pull request be updated to cover .net 4.7 as well which will only have buildtools 15?